.changeset/fair-needles-pull.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+build(deps): bump typescript from 5.2.2 to 5.4.5
```

Comment:
Interesting that it already [bumped](https://github.com/FuelLabs/fuels-ts/blob/ad9adcd0a9dc9851f3b9b5ba790954e979d02f0c/package.json#L110) typescript to 5.4.5 a month ago but is only now bumping the typescript+eslint-related dependencies.

Reply:
I still don't understand how these PRs have no changes to a `package.json`, only the lock file.

As soon as someone updates the lock file, it can be easily lost.

Reply:
It's also downgrading for some dependencies?
<img width="653" alt="image" src="https://github.com/user-attachments/assets/3a492197-9d28-400b-a0b7-9b7e10538593">